classes/item.rb

```diff
@@ -0,0 +1,29 @@
+require 'securerandom'
+require 'date'
+
+class Item
+  attr_accessor :genre, :author, :label, :publish_date
+  attr_reader :id, :archived
+
+  def initialize(publish_date)
+    @id = id || SecureRandom.uuid
+    @publish_date = publish_date
+    @archived = false
+  end
+
+  def move_to_archive
+    @archived = true if can_be_archived?
+  end
+
+  def add_genre(genre)
+    genre.items << self unless genre.items.include?(self)
+    @genre = genre
+  end
+
```

Comment:
- [ ] It's great that you have implemented some tests for this class but as suggested by the previous reviewer multiple times that all the public methods should be tested 🤔 so it would be great if you test the add_genre and move_to_archive methods to make your project follow the requirements and verify that your app is working without errors 👍